blogs/26.08.md

```diff
@@ -0,0 +1,62 @@
+# Wiedereinstieg in Phython
+
+## Erste Schritte
+Als erstes habe ich Funktionen wie 'pencolor', 'pensize', 'dot', 'forward' wieder gebraucht. Da sind mir schon wieder die meisten Sachen eingefallen und die Aufgaben waren auch gar kein Problem.
+
+## Füllen und wiederholen
+Danach habe ich mir den weiterführenden Begriff zum wiederholen angeschaut:
+````
+for i in range():
+````
```

Comment:
3 "Backticks" ergeben einen Codeblock. Zusätzlich können Sie die Dateiendung der Datei verwenden, um Codehighlighting zu aktivieren, hier also `.py`

````suggestion
```py
for i in range():
```
````